src/CandidateList.js

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+function CandidateList() {
+    const [candidates, setCandidates] = useState(null);
+    async function getCandidates() {
+        var data = await axios.get(
+            'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+        );
+        setCandidates(data.data);
```

Comment:
1. `var`보다는 `const`로 쓸 수 있는 상황에는 `const`로 선언하여 쓰는게 좋을 것 같아요!
[참고](https://happycording.tistory.com/entry/let-const-%EB%9E%80-%EC%99%9C-%EC%8D%A8%EC%95%BC%EB%A7%8C-%ED%95%98%EB%8A%94%EA%B0%80-ES6)

2. 그리고 axios로 받아온 response를 이렇게 비구조화할당해서 받을수도있어요!
```suggestion
        const {data}= await axios.get(
            'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
        );
        setCandidates(data);
```

Reply:
아 const를 왜 써야하는지 잘 몰랐는데 이제 알 것 같아요!! 비구조화할당해서 받아올수도 있군용 ㅎㅎ